ci/order-crates-for-publishing.py

```diff
@@ -21,6 +21,36 @@ def load_metadata():
     return json.loads(subprocess.Popen(
         cmd, shell=True, stdout=subprocess.PIPE).communicate()[0])
 
+no_explicit_version_specified = '*'
+
+def is_special_cased_self_dep(package, dependency):
+    is_special_cased = False
+
+    # Sometimes self-dev-dep with dev-context-only-utils is needed and this dep
```

Comment:
The use of "self-dev-dep" in this comment throws me off. Maybe it can be expanded or more context added.

It looks like ` is_special_cased_self_dep` only cares about the `dev-context-only-util` crate, so I'd rather it be more explicit in the function name that this is really only about that one crate

Reply:
haha, no wonder for throwing you off. the proper commenting resulted in quite a high wall of text: 069cefafd6ca5ffb56c142d4ae88c6d0f8754257

yeah, i needed fresh eyes to spell the hidden context around `dev-context-only-utils` out. thanks for review. :)